org.osgi.test.junit5/src/main/java/org/osgi/test/junit5/bundle/BundleState.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) OSGi Alliance (2019, 2020). All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.osgi.test.junit5.bundle;
```

Comment:
I think this one belongs in common, not junit5. Possibly integrated with the existing classes in three bitmap subpackage.

Reply:
I removed the Enum. The static fields in Bundle are descriptive enough

Reply:
One advantage the enum does have is compile time type safety. But then it's also supposed to be a bitmask so you can OR values together which you couldn't do with an enum. So you probably made the right call here.